xep-0425.xml

```diff
@@ -84,30 +97,29 @@
 </message>]]></example>
 
   <p>The moderator sees the message and indicates to their client that the message should be retracted. The client then sends an IQ stanza to the MUC service, requesting that the message be retracted.</p>
+  <p>An optional &lt;reason/&gt; element may be included inside the &lt;moderate/&gt; element, to provide a human-readable explanation why the message was retracted.</p>
 
   <example caption="The moderator's client sends an IQ stanza to the MUC service"><![CDATA[
 <iq type='set' to='room@muc.example.com' id='retract-request-1'>
-  <apply-to id="stanza-id-1" xmlns="urn:xmpp:fasten:0">
-    <moderate xmlns='urn:xmpp:message-moderate:0'>
-      <retract xmlns='urn:xmpp:message-retract:0'/>
-      <reason>This message contains inappropriate content for this forum</reason>
-    </moderate>
-  </apply-to>
+  <moderate id="stanza-id-1" xmlns='urn:xmpp:message-moderate:1'>
+    <retract xmlns='urn:xmpp:message-retract:1'/>
+    <reason>This message contains inappropriate content for this forum</reason>
+  </moderate>
 </iq>]]></example>
 
   <section2 topic='Success case' anchor='usecase-success'>
 
   <p>If the moderator is allowed to moderate the message, the groupchat service will send a message stanza to all participants (including the moderator), indicating that the message has been retracted and by whom.</p>
```

Comment:
Is this the only protection against a non-moderator issuing a retraction? If so it's not enough because the MUC may not understand this protocol and therefore won't be able to perform this filtering.

Regardless, how, when, and where retractions are supposed to be validated should be in the security considerations section.

Reply:
> To prevent message spoofing, it's very important to check that the moderation message comes from the MUC service 

its already in the security considerations, this is enough, don’t you think?